tasks/createPosition/index.ts

```diff
@@ -83,39 +79,43 @@ task('createPosition', 'Create stETH position on AAVE')
       AAVELendigPoolABI,
       config.provider,
     )
+
+    config.provider.on(aaveLendingPool.filters.Deposit(ADDRESSES.main.DAI), (log, event) => {
+      console.log('Deposit event', log)
+      console.log('Deposit event', event)
+    })
+
     const aaveDataProvider = new hre.ethers.Contract(
       ADDRESSES.main.aave.DataProvider,
       AAVEDataProviderABI,
       config.provider,
     )
-
-    const aavePriceOracle = new hre.ethers.Contract(
-      ADDRESSES.main.aavePriceOracle,
-      aavePriceOracleABI,
-      config.provider,
-    )
-
+    new hre.ethers.Contract(ADDRESSES.main.aavePriceOracle, aavePriceOracleABI, config.provider)
```

Comment:
Either I'm blind or this does absolutely nothing. I mean it creates an instance and destroys it because nothing uses it.